lib/adf_info.py

```diff
@@ -211,15 +228,15 @@ def __init__(self, config_file, debug=False):
                     msg += f"{data_name}, using first found year: {found_eyear_baseline}\n"
                     print(msg)
                     eyear_baseline = found_eyear_baseline
-            #End if
-
-            #Check if history file path exists:
-            if baseline_hist_locs:
+            # End if
 
+            # Check if history file path exists:
+            if any(baseline_hist_locs):
+                hist_str = baseline_hist_str[0]
```

Comment:
This fails if the user only supplies a string, should it be run through the same function as the test case(s) using `hist_str_to_list` or do it inline 

```suggestion
                if not isinstance(conf_val, list):
                    baseline_hist_str = [baseline_hist_str]
                hist_str = baseline_hist_str[0]
```

or above where `baseline_hist_str` is declared?

Reply:
Justin, I'm having trouble getting the changes you sent me to merge. I'm assuming this suggestion was included before you did the wider merge?